v3/src/components/graph/components/background.tsx

```diff
@@ -123,6 +124,7 @@ export const Background = forwardRef<SVGGElement, IProps>((props, ref) => {
         (enter) => {
           enter.append('rect')
             .attr('class', 'graph-background')
+            // @ts-expect-error strictFunctionTypes
```

Comment:
FWIW: in CLUE, this wasn't needed after I changed line 109 to `dragBehavior = drag<SVGRectElement, number>()`.